FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Firebase 7.1.0
+- [fixed] `+[FIRApp registerInternalLibrary:withName:]` takes too much time. (#6902)
```

Comment:
nit: Prefer Swift in release notes. Also updated wording a bit

```swift
- [fixed] Reduced `FirebaseApp.registerInternalLibrary(name:)` impact on app launch time.
```

Reply:
`+[FIRApp registerInternalLibrary:withName:]` is an internal method that is not available from Swift so using Swift version would be rather confusing. I applied the wording change and added the affected public API mention. PTAL